### PR TITLE
Release 10.5.6

### PR DESCRIPTION
**Pull request by swethakundaram**

### CODE REVIEW

File: learn/wavemaker-release-notes/v10-5-6.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+## Bug Fixes
+
+---
+
+- Fixed issue related to exporting project shell to EDN.
+- Fixed issues related to editing and displaying the user profile name.
+- Fixed issue related to default messages which are not localized.
+- Fixed issue related to the **onSuccess** handler for file upload variable to customize the upload success notification.
+- Fixed issue related to the `reset()` method that was not working for an HTML upload widget.
+- Fixed issue related to accessing the second level Partial page widgets from `App.activePage`.
```
Comment (vibhus) [inline learn/wavemaker-release-notes/v10-5-6.md:24]: Can be changed to "Fixed issue related to accessing the second level Partial page widgets from `App.activePage` when the partial is loaded in a page dialog"

File: learn/wavemaker-release-notes/v10-5-6.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+- Update of Android **targetSdkVersion** is required by November 2, 2020. For more information, see [Mobile Build - Manual](learn/hybrid-mobile/mobile-build-manual).
+
+## Bug Fixes
+
+---
+
+- Fixed issue related to exporting project shell to EDN.
+- Fixed issues related to editing and displaying the user profile name.
+- Fixed issue related to default messages which are not localized.
+- Fixed issue related to the **onSuccess** handler for file upload variable to customize the upload success notification.
```
Comment (vibhus) [inline learn/wavemaker-release-notes/v10-5-6.md:22]: Should be changed to "When onSuccess handler for a service variable used to upload file is overridden, the default success toaster is now suppressed. This allows users to customise the behaviour as per need."

File: learn/wavemaker-release-notes/v10-5-6.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+id: ""
+sidebar_label: "Release v10.5.6"
+---
+*WaveMaker released a new update version with bug fixes.*
+
+## Improvements
+
+---
+
+- Stackable multiple toaster messages. This offers a way to display multiple toasters where you can configure the time duration.
```
Comment (vibhus) [inline learn/wavemaker-release-notes/v10-5-6.md:12]: "Stackable multiple toaster messages. This offers a way to display multiple toasters in a page and can be configured via a simple method." We also need to create a how to doc for it.

File: learn/wavemaker-release-notes/v10-5-6.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+- Update of Android **targetSdkVersion** is required by November 2, 2020. For more information, see [Mobile Build - Manual](learn/hybrid-mobile/mobile-build-manual).
+
+## Bug Fixes
+
+---
+
+- Fixed issue related to exporting project shell to EDN.
+- Fixed issues related to editing and displaying the user profile name.
+- Fixed issue related to default messages which are not localized.
```
Comment (vibhus) [inline learn/wavemaker-release-notes/v10-5-6.md:21]: "Few messages which were not localized are now localized."

File: learn/wavemaker-release-notes/v10-5-6.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+- Fixed issue related to the **onSuccess** handler for file upload variable to customize the upload success notification.
+- Fixed issue related to the `reset()` method that was not working for an HTML upload widget.
```
Comment (vibhus) [inline learn/wavemaker-release-notes/v10-5-6.md:23]: Change to, "On clicking reset on a form, the file widget inside it was not getting reset. This issue is now fixed"

File: learn/wavemaker-release-notes/v10-5-6.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+- Fixed issue related to accessing the second level Partial page widgets from `App.activePage`.
+- Fixed issue related to the **Animation Interval** property for the **Carousel** widget to set it through the script.
+- Fixed issue related to the **Chips** widget dataset that is not updating the UI when the variable is updated.
+
+### Data Table
+
+- Fixed issue related to DataTable column styles that are given in %.
+- Fixed issue related to the **INFO** toaster for Inline Edit in the DataTable row.
+- Fixed issues related to applying stypes for Data Table, including "table-bordered" and "table-striped" not migrating from 9.x to 10.x version.
```
Comment (vibhus) [inline learn/wavemaker-release-notes/v10-5-6.md:32]: This is not in 10.5.6. Please remove it

File: learn/wavemaker-release-notes/v10-5-6.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+- Fixed issue related to the pagination based on the **Records per Request** property in the Data Table.
```
Comment (vibhus) [inline learn/wavemaker-release-notes/v10-5-6.md:33]: No fix has been done for this issue as it is by design. Please remove it.

File: learn/wavemaker-release-notes/v10-5-6.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+
```
Comment (vibhus) [inline learn/wavemaker-release-notes/v10-5-6.md:34]: More bug fixes that can be added:
* Fixed scrolling issues with content inside a design dialog.
* Fixed few broken documentation links in the studio.

**Review by vibhus — COMMENTED** [review]
Please make the suggested changes

File: learn/wavemaker-release-notes/v10-5-6.md [hunk 9]
```diff
@@ -18,19 +18,14 @@ sidebar_label: "Release v10.5.6"
...
 - Fixed issue related to the **Animation Interval** property for the **Carousel** widget to set it through the script.
 - Fixed issue related to the **Chips** widget dataset that is not updating the UI when the variable is updated.
-
-### Data Table
-
-- Fixed issue related to DataTable column styles that are given in %.
-- Fixed issue related to the **INFO** toaster for Inline Edit in the DataTable row.
-- Fixed issues related to applying stypes for Data Table, including "table-bordered" and "table-striped" not migrating from 9.x to 10.x version.
-- Fixed issue related to the pagination based on the **Records per Request** property in the Data Table.
+- Fixed scrolling issues with content inside a design dialog.
```
Comment (vibhus) [inline learn/wavemaker-release-notes/v10-5-6.md:27]: These two should go under Bug Fixes. They are not related to Table widget

Comment (swethakundaram) [reply learn/wavemaker-release-notes/v10-5-6.md:27]: @vibhus which two?

Comment (vibhus) [reply learn/wavemaker-release-notes/v10-5-6.md:27]: OK, I misunderstood the change. Looks fine.

**Review by vibhus — APPROVED** [review]
Looks good

**Pull request merged**